server.js

```diff
@@ -1,17 +1,12 @@
-const express = require("express");
+require("dotenv").config();
 
+const express = require("express");
 const crawlRoute = require("./routes/crawlRoute.js");
-
 const app = express();
-
-const port = 3000;
-
-app.get("/", (req, res) => {
-  res.send("get test");
-});
+const port = process.env.port || 3000;
 
 app.use("/crawl", crawlRoute);
 
 app.listen(port, () => {
-  console.log(`Example app listening on port ${port}`);
+  console.log(`Crawling server start on port ${port}`);
```

Comment:
```suggestion
  console.log(`Crawling server start on port ${port}`);
```
단순히 콘솔을 찍기위한 부분으로 보입니다. 이 부분이 꼭 필요한 걸까요?

Reply:
서버 개발 초기 "node server.js"명령어로 실행 시, 서버가 떠있는지 확인할 수 없고 포트 정보를  확인하는 용도로 썼지만, 현재는 서버 자동 재시작 모듈이 설치되어 있어서 꼭 필요하지 않습니다. 수정하겠습니다!